### PR TITLE
Home Assistant Shim

### PR DESCRIPTION
**Pull request by maddox**
This is a first pass at a shim for [Home Assistant](https://home-assistant.io).

Right now this only supports lights, but I'd like it to at least manage switches too before it lands.

//cc @nfarina @balloob

### CODE REVIEW

**Comment by maddox** [comment]
Before this lands, I'd like to have it support switches too. But I'm not exactly sure how to support 2 types of accessories inside here. I'd love some suggestions from anyone that might have done this in another shim. I couldn't really tell if any other shim has done that :/

File: platforms/HomeAssistant.js [hunk 1]
```diff
@@ -0,0 +1,301 @@
...
+    })
+
+  }
+}
+
+function HomeAssistantAccessory(log, data, client) {
+  // device info
+  this.data = data
+  this.entity_id = data.entity_id
+  this.name = data.attributes.friendly_name
```
Comment (balloob) [inline platforms/HomeAssistant.js]: Not every entity will have this attribute. In [the HA frontend](https://github.com/balloob/home-assistant-js/blob/master/src/modules/entity/model.js#L40) I fall back to the object_id (part right of the `.` in entity_id) with `_` replaced by `<space>`

Comment (maddox) [inline platforms/HomeAssistant.js]: OK awesome, good to know. thanks.

**Comment by maddox** [comment]
Woot woot, this is ready to go!

**Comment by maddox** [comment]
Switch got added.

**Pull request merged**